src/Commands/MigrateFresh.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Stancl\Tenancy\Commands;
+
+use Illuminate\Console\Command;
+use Illuminate\Database\Console\Migrations\MigrateCommand;
+use Illuminate\Database\Migrations\Migrator;
+use Stancl\Tenancy\DatabaseManager;
+use Stancl\Tenancy\Traits\DealsWithMigrations;
+use Stancl\Tenancy\Traits\HasATenantsOption;
+
+class MigrateFresh extends MigrateCommand
```

Comment:
I'll review this thoroughly later, but now one thing I'd do is make this class final:
```suggestion
final class MigrateFresh extends MigrateCommand
```
@antonkomarev pointed out that
> adding new options to command is not a breaking change

That is assuming users won't extend this class. So for now, I'd make it final.